app/services/food_service.rb

```diff
@@ -0,0 +1,14 @@
+class FoodService
+  def self.search_foods_by_ingredient(ingredient)
+    response = conn.get("/search?query=#{ingredient}")
+    response_json = JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.conn
+    Faraday.new('https://api.nal.usda.gov/fdc/v1/foods') do |faraday|
+      faraday.params['api_key'] = ENV['FOODS_API_KEY']
+    end
+  end
```

Comment:
To create the connection, leave the endpoint off. You should just be connecting to `https://api.nal.usda.gov` here, and then you can make the get request in the `search_foods_by_ingredient` method to `/fdc/v1/foods/search?query=#{ingredient}`